faq.md

```diff
@@ -108,29 +108,27 @@ $ export PATH=~/.local/bin:$PATH
 $ reuse --help
 ```
 
-## How do I exclude a file from REUSE compliance testing? {#exclude-file}
+## Exclude a file from REUSE compliance testing {#exclude-file}
 
-If the file is a build artifact and you use Git, simply make sure that the file
+If the file is a build artifact and you use Git, ensure the file
 is covered by your `.gitignore` file.
 
 Otherwise, you cannot exclude files from REUSE compliance testing. It would go
-entirely against the purpose of REUSE: making sure that every file has a
+entirely against the purpose of REUSE: to make sure every file has a
 copyright and license tag.
 
-If you really want to exclude a file, consider using the
-[CC0](https://creativecommons.org/publicdomain/zero/1.0/) license for this
-file. By doing this, you put the file in the public domain, or your country's
```

Comment:
Or you do nothing of the sort, which is why this is bad advice.
Much less do you do anything to help yourself and your newfound assumptions if you want to have other contributors from other jurisdictions.

Reply:
Thanks for the wording improvements! However, I'd suggest to not change this content bit, but concentrate on wording/typos in this PR.

Reply:
@mxmehl That part is no less a mistake. Licensing something CC0 doesn't put it into the public domain. Much less in any sense compatible with taking contributions globally. This is because some countries don't have this provision.
CC licensing is furthermore a scourge meaningless in terms of reuse, because it doesn't necessitate anything meaningful since NC is part of it.

Reply:
> Licensing something CC0 doesn't put it into the public domain. Much less in any sense compatible with taking contributions globally.This is because some countries don't have this provision.

The fallback provisions of CC0 are functionally identical to the public domain, however.

> [A person using CC0 (called the “affirmer” in the legal code) dedicates a work to the public domain by waiving all of his or her copyright and neighboring and related rights, if any, in a work, to the fullest extent permitted by law. If the waiver isn’t effective for any reason, then CC0 acts as a license from the affirmer granting the public an unconditional, irrevocable, non exclusive, royalty free license to use the work for any purpose.](https://wiki.creativecommons.org/wiki/CC0_FAQ#How_does_it_work.3F)

And I'll happily defer to Creative Commons as an expert authority when they say:

> [And while no tool, not even CC0, can guarantee a complete relinquishment of all copyright and database rights in every jurisdiction, we believe it provides the best and most complete alternative for contributing a work to the public domain given the many complex and diverse copyright and database systems around the world.](https://creativecommons.org/share-your-work/public-domain/cc0/)

I'm not sure exactly how it's a mistake to suggest CC0 as a "no rights reserved" solution. Are there jurisdictions where CC0 is broken?

Reply:
It doesn't do anything productive. What it does is proliferate the amount of licenses to take into account, where this one in particular adds nothing over its already mentioned alternatives. I would argue it opens up a whole new FAQ:

>to the fullest extent permitted by law.

This isn't "complex". You have to adhere to a copyright system and its protections to license your way out of it, and that doesn't work. Reason why is some jurisdictions require commission, some don't allow it, and some allow changing ones will (or ones next of kin to do so). Some tack on extra requirements to avoid slander by default, etc. It doesn't reliable remove one person from the equation, much less function any differently between many.

"we believe it provides the best and most complete alternative for contributing a work to the public domain" isn't "If you really want to exclude a file" because no additional benefit is to be had here.

Through no fault of the license is that the case, but CC remains an incompatible mess at large.

Reply:
We already have at least one issue suggestion to re-word this. Please let's discuss this over there.